### ISSUE TITLE
Issue stop sync blockchain after upgrade 2.3.3 to 2.3.5

### ISSUE THREAD

**Issue by angelocastiglione** [DeFiCh/app#852]
After install upgrade to 2.3.5 the network go to error sto to sync and corrupt database of blockchain

[log.zip](https://github.com/DeFiCh/app/files/6471899/log.zip)

- DeFi Wallet Version: 2.3.5
- Operating System (OS): Windows 10
– Current free storage capacity: 22GB

**Comment by lraphael** [comment]
The same thing happened to me with version 2.3.4 and 2.3.5. At this moment, the blockchain no longer synchronises and no more logs are written. If it is then blocked and I try to enter a command in the console, it is also not executed. I confirmed with Enter, but it hangs. See screenshot.

![Bildschirmfoto 2021-05-13 um 13 59 23](https://user-images.githubusercontent.com/2973257/118124583-14f72500-b3f6-11eb-99d2-6e5629904516.png)

I then tried to close the program. The shutdown was then also forced, as it was added in the last patch: https://github.com/DeFiCh/app/pull/851/files

However, it did not succeed. Even a `kill -15 defidPID` can no longer terminate the program. 

![Bildschirmfoto 2021-05-13 um 14 11 55](https://user-images.githubusercontent.com/2973257/118124972-a797c400-b3f6-11eb-94a2-577418465707.png)
![Bildschirmfoto 2021-05-13 um 14 15 05](https://user-images.githubusercontent.com/2973257/118125000-b2eaef80-b3f6-11eb-88dd-bca83dc4090b.png)

Attached the traceback from Task Manager:
[defid_traceback.txt](https://github.com/DeFiCh/app/files/6472402/defid_traceback.txt)


OS: Mac OS X 11.3.1 Big Sur

**Comment by angelocastiglione** [comment]
I have delete the blockchain and downloaded again with the snapshot. After syncronized I have again the same issue, the app freeze, and not sync 

[2021-05-13 17:56:20.717] [IPC-RENDERER-error] [isBlockchainStarted] "Request failed with status code 500"

[log.log](https://github.com/DeFiCh/app/files/6473651/log.log)

**Comment by angelocastiglione** [comment]
> I have delete the blockchain and downloaded again with the snapshot. After syncronized I have again the same issue, the app freeze, and not sync
> 
> [2021-05-13 17:56:20.717] [IPC-RENDERER-error] [isBlockchainStarted] "Request failed with status code 500"
> 
> [log.log](https://github.com/DeFiCh/app/files/6473651/log.log)

After wait few minute the sync start again, and the app working again and not damage the blockchain, is most important in this case to dont close the app if the sync is stopped and waiting to run again

**Comment by angelocastiglione** [comment]
![Schermata da 2021-05-13 21-37-55](https://user-images.githubusercontent.com/82758166/118177872-915c2900-b433-11eb-9c03-bae14c1287be.png)

This show when the sync go to error and not working.

**Comment by izzycsy** [comment]
Hi @angelocastiglione , thanks for your feedback, it seems like you've insufficient disk space. Currently the app requires 120GB of storage excluding temporary storage should you apply snapshot (40.51GB). Kindly clear storage and try again or wait for the Light Wallet. Please keep your `wallet.dat` safe.   

Hi @lraphael , can you check your storage capacity please? 

Regarding the issue [`Insufficient Storage`](https://github.com/DeFiCh/app/issues/864), we can further discuss on the issue there. We would like to keep same issues in one thread. Thank you.

**Comment by lraphael** [comment]
@izzycsy For me, the problem has nothing to do with the storage space. I have the blockchain on an external hard drive with more than 500G of free space. The problem also happens when I start the node directly, without using the app. The problem has also been posted by several users on Telegram. I hope it works with Node 1.7 and currently stay with the app on 2.3.3 and Node on 1.6.2.
I tried updating again yesterday but the same problem persists.

Earlier in this ticket I also posted the traceback and that kill -15 defidPID does not work. This has nothing to do with hard disk space.
